src/Command/ConfigureCommand.php

```diff
@@ -27,16 +27,24 @@
 
 class ConfigureCommand extends Command
 {
-    /**
-     * @var ConsoleHelper
-     */
-    private $consoleHelper;
+    protected function configure()
```

Comment:
don't ever have a plan to add anything there and not sure what we need to add :)

Reply:
I mean to add `@inherit` doc blocks when the method has a parent one; but nvm if you don't like that practice. I kinda like it because you can easily tell if it has a parent one without comparing the parent class but that's it